providers/labels.go

```diff
@@ -0,0 +1,79 @@
+package providers
+
+import "strings"
+
+// Label keys
+const (
+	LabelPrimaryCategory     = "primary-category"
+	LabelSecondaryCategories = "secondary-categories"
+)
+
+// Label values (categories)
+const (
+	CategoryAdNetworks                  = "Ad Networks"
+	CategoryAI                          = "AI"
+	CategoryAnalytics                   = "Analytics"
+	CategoryAnalyticsBI                 = "Analytics & BI"
+	CategoryAppointmentScheduling       = "Appointment Scheduling"
+	CategoryBilling                     = "Billing"
+	CategoryCDP                         = "CDP"
+	CategoryCloudContentCollaboration   = "Cloud Content Collaboration"
+	CategoryCodeManagement              = "Code Management"
+	CategoryCommunication               = "Communication"
+	CategoryContactCenter               = "Contact Center"
+	CategoryContractLifecycleManagement = "Contract Lifecycle Management"
+	CategoryConversationalPlatform      = "Conversational Platform"
+	CategoryCPQ                         = "CPQ"
+	CategoryCRM                         = "CRM"
+	CategoryCustomerDataPlatform        = "Customer Data Platform"
```

Comment:
@RajatPawar can you merge "CDP" and "Customer Data Platform"?

Reply:
Oops, just lifted these off Airtable, fixing it now